kernel/packages/entryPoints/unity.ts

```diff
@@ -58,26 +69,22 @@ initializeUnity(container)
     })
     i.ConfigureHUDElement(HUDElementID.AIRDROPPING, { active: true, visible: true })
     i.ConfigureHUDElement(HUDElementID.TERMS_OF_SERVICE, { active: true, visible: true })
-    //NOTE(Brian): Scene download manager uses meta config to determine which empty parcels we want
-    //             so ensuring meta configuration is initialized in this stage is a must
-    //NOTE(Pablo): We also need meta configuration to know if we need to enable voice chat
-    await ensureMetaConfigurationInitialized()
-
-    const voiceChatEnabled = isVoiceChatEnabled(globalThis.globalStore.getState())
 
-    i.ConfigureHUDElement(HUDElementID.TASKBAR, { active: true, visible: true }, { enableVoiceChat: voiceChatEnabled })
-    i.ConfigureHUDElement(HUDElementID.WORLD_CHAT_WINDOW, { active: true, visible: true })
     i.ConfigureHUDElement(HUDElementID.OPEN_EXTERNAL_URL_PROMPT, { active: true, visible: false })
     i.ConfigureHUDElement(HUDElementID.NFT_INFO_DIALOG, { active: true, visible: false })
     i.ConfigureHUDElement(HUDElementID.TELEPORT_DIALOG, { active: true, visible: false })
-    i.ConfigureHUDElement(HUDElementID.CONTROLS_HUD, { active: true, visible: false })
-    i.ConfigureHUDElement(HUDElementID.EXPLORE_HUD, { active: true, visible: false })
-    i.ConfigureHUDElement(HUDElementID.HELP_AND_SUPPORT_HUD, { active: true, visible: false })
-    i.ConfigureHUDElement(HUDElementID.USERS_AROUND_LIST_HUD, { active: voiceChatEnabled, visible: false })
+
+    //NOTE(Brian): Scene download manager uses meta config to determine which empty parcels we want
+    //             so ensuring meta configuration is initialized in this stage is a must
+    //NOTE(Pablo): We also need meta configuration to know if we need to enable voice chat
```

Comment:
Please let's stop adding these notes with `@authors` -- it's very prone to aging badly.